services/nutanix-ai/2.0.0/kustomization.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
+  - ./pre-install
```

Comment:
ideally kserve should also be moved to pre-install? why is this one only special? can we keep in teh same dir as kserve?

Reply:
I will move to preinstall folder later today in another PR